src/jobservice/job/impl/scan/all.go

```diff
@@ -95,7 +95,7 @@ func (sa *All) Run(ctx job.Context, params job.Parameters) error {
 				data, err := ioutil.ReadAll(resp.Body)
 				if err != nil {
 					logger.Errorf("Failed to read response, error: %v", err)
-				} else if resp.StatusCode != http.StatusOK {
+				} else if resp.StatusCode/100 != 2 {
```

Comment:
According to the `Scanner Adapter API` in the proposal,  the status code is `StatusAccepted`( 202 ) when submit scan request success and the `/api/repositories/*/tags/:tag/scan` also return `StatusAccepted` when success.

Reply:
I know that, to avoid changing again. So, I use a general way.

Reply:
let's use 202 replace it